lms/djangoapps/verify_student/services.py

```diff
@@ -122,12 +122,7 @@ def get_expiration_datetime(cls, user, statuses):
         sso_id_verifications = SSOVerification.objects.filter(**filter_kwargs).order_by('-' + sort_by)
         manual_id_verifications = ManualVerification.objects.filter(**filter_kwargs).order_by('-' + sort_by)
```

Comment:
does this really need to be sorted anymore?

Reply:
sure doesn't